composer.json

```diff
@@ -13,6 +13,7 @@
     "require": {
         "php": "^8.1",
         "jms/serializer-bundle": "^3.10 || ^4.2 || ^5.4",
+        "symfony/dependency-injection": "^5.4 || ^6.0 || ^7.0",
         "symfony/expression-language": "~3.0 || ~4.0 || ~5.0 || ~6.0 || ~7.0",
```

Comment:
would it make sense to drop support for symfony 3 and 4 completely then? I mean they are long dead anyway

Reply:
I'm also proposing to remove it for https://github.com/willdurand/Hateoas/pull/343 so it wouldn't make sense to support it here then. But maybe let's wait for a maintainer